docs/user/resources/01-serverless.md

```diff
@@ -0,0 +1,89 @@
+---
```

Comment:
Please rename the doc file to `06-20-serverless-cr.md`

Reply:
done